### PR TITLE
Fix : Sign In Page Style

### LINKED ISSUE #31: Adjust the visual design of the admin login form
Shown below is a screenshot of the admin login form.

LiveWebServer-Admin-11-21-2024_06_03_PM

* Make sure the height of the navigation bar is the same as the height of the navigation bar after login.
* Place the text “LiveWebServer Admin” in the area boxed in red.
* Delete the gray rectangle in the area boxed in green.

For your information, the following is a screenshot of the navigation bar of the administration screen after login.

LiveWebServer-Admin-11-21-2024_06_02_PM

### PR DESCRIPTION
**Pull request by k70suK3-k06a7ash1**
- resolve #31

### CODE REVIEW

**Comment by k70suK3-k06a7ash1** [comment]
@kuroda 
please check this PR

THX

File: lib/live_web_server_web/components/layouts/app.html.heex [hunk 1]
```diff
@@ -42,6 +42,18 @@
       </dialog>
     </nav>
   <% end %>
+
+  <%= unless @current_administrator do %>
+    <nav phx-hook="ShowModalSignOut" id="live-web-server-header" class="flex ml-4">
+      <div class="flex flex-wrap content-center">
+        <span class="text-2xl font-bold mr-4">LiveWebServer Admin</span>
+  
+
```
Comment (kuroda) [inline lib/live_web_server_web/components/layouts/app.html.heex]: Remove the extra line breaks.

File: lib/live_web_server_web/components/layouts/app.html.heex [hunk 2]
```diff
@@ -42,6 +42,18 @@
...
+  <%= unless @current_administrator do %>
+    <nav phx-hook="ShowModalSignOut" id="live-web-server-header" class="flex ml-4">
+      <div class="flex flex-wrap content-center">
+        <span class="text-2xl font-bold mr-4">LiveWebServer Admin</span>
+  
+      
+      </div>
+  
+
```
Comment (kuroda) [inline lib/live_web_server_web/components/layouts/app.html.heex]: Remove the extra line breaks.

**Review by kuroda — CHANGES_REQUESTED** [review]
See my comments

**Pull request closed without merging**